week-1/errors/2.js

```diff
@@ -1,5 +1,10 @@
 // Currently trying to print the string "I was born in Bolton" but it isn't working...
 // what's the error ?
-
-console.log(`I was born in ${cityOfBirth}`);
 const cityOfBirth = "Bolton";
+console.log(`I was born in ${cityOfBirth}`);
+
+
+
+// ReferenceError: Cannot access 'cityOfBirth' before initialization
```

Comment:
Another good explanation - I like the fact that you've also logged the error here too!